docs/dg/workflow.md

```diff
@@ -17,8 +17,9 @@ Our workflow is mostly based on the guidelines given at se-education.org/guides.
 
 * As we squash the commits when merging a PR, there is ==no need to follow a strict commit organization or write elaborate commit messages for each commit==.<br>
   However, when pushing new commits to your PR branch, do clean up _new_ commits (i.e., commits not yet pushed) e.g., delete temporary print statements added for debugging purposes.
-* You can refer to the [Architecture](architecture.html) and the [HTML Report](report.html) to learn about the design and implementation of RepoSense.
-* The sections below has more information about various stages of submitting a PR.
+* Although it is no necessary to write elaborate commit message, it is still recommended for each PR to contain a brief commit message in its description. The guideline is available [here](https://se-education.org/guides/guidelines/PRs.html).
```

Comment:
```suggestion
* Although it is unnecessary to write a detailed commit message, each PR is still recommended to contain a brief commit message in its description. The guideline is available [here](https://se-education.org/guides/guidelines/PRs.html).
```

Reply:
>Although it is unnecessary to write a detailed commit message, each PR is still recommended to contain a brief

I think this is misleading. The proposed commit message for a PR should be complete, not brief.

Reply:
Ya agreed. How about - **There should be a proposed commit message for each PR. Although the suggested message does not have to be detailed, it should be complete. The guideline.........** - Since based on our guidelines it is compulsory to have a proposed commit message and not recommended exactly.

Reply:
>Although the suggested message does not have to be detailed

We shouldn't omit any relevant details :-). Of course we shouldn't include _irrelevant_ details but that goes without saying. In other words, there is no need for this qualifier.

Any yes, it is compulsory, not merely _recommended_. Something like this?

>* In the PR description, please propose a commit message to be used when the PR is merged eventually. The commit message should follow the guidelines given [here](). You may refer to [this PR]() for an example.

Reply:
@HCY123902 Can move this change to a new PR based on Prof's recommendation. Then I will review it again.

Reply:
Sure, no problem.